src/Scripting/Core/Resolvers/RuntimeMetadataReferenceResolver.cs

```diff
@@ -46,24 +39,26 @@ internal sealed class RuntimeMetadataReferenceResolver : MetadataReferenceResolv
 
         public override ImmutableArray<PortableExecutableReference> ResolveReference(string reference, string baseFilePath, MetadataReferenceProperties properties)
         {
-            if (PathResolver != null && PathUtilities.IsFilePath(reference))
+            if (PackageResolver != null)
```

Comment:
I think it would be clearer to parse up front and then try either nuget resolution or file resolution.  Consider adding a TryParse method that returns a refined string to be passed to ```ResolveNuGetPackage```.